### PR TITLE
Updating project community information

### PR DESCRIPTION
**Pull request by tsurdilo**
This pr updates the community information and structures the community and governance info.

### CODE REVIEW

File: workflow/spec/readme.md [hunk 1]
```diff
@@ -27,20 +27,20 @@ The following documents are available:
...
+Mailing list for e-mail communications:
 
 - Send emails to: [cncf-wg-serverless](mailto:cncf-wg-serverless@lists.cncf.io)
-- To subscribe see: https://lists.cncf.io/g/cncf-wg-serverless
-
-And a #serverless-workflow Slack channel:
-[https://slack.cncf.io/](https://slack.cncf.io/)
+- To subscribe see: [https://lists.cncf.io/g/cncf-wg-serverless](https://lists.cncf.io/g/cncf-wg-serverless)
 
+Community Slack channel: #serverless-workflow - [https://slack.cncf.io/](https://slack.cncf.io/)
```
Comment (cathyhongzhang) [inline workflow/spec/readme.md]: Better to change to: 
Email communication: 
Subscription: 
Community Slack Channel:

Comment (cathyhongzhang) [reply workflow/spec/readme.md]: Quite some links are not accessible (readme, owner, etc. ). Could you fix these and reload the updated one? Thanks.

Comment (tsurdilo) [reply workflow/spec/readme.md]: fixed the communication changes requested. 
@cathyhongzhang not sure what is missing, you can see all the changes as they would appear if pr was merged here: 

https://github.com/tsurdilo/wg-serverless/tree/community/workflow/spec

If you find some specific links not working please let me know to fix.

File: workflow/spec/community/open-source.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Serverless Workflow - Open Source
+
+If you have created (or know of) an open source library or product that uses 
+Serverless Workflow, please include it in the list below.
+
+- [workflow-api](https://github.com/serverless-workflow/workflow-api): Java based api/spi of the Serverless Workflow specification. Adds POJO generation from specification JSON schema.
+- [workflow-impl](https://github.com/serverless-workflow/workflow-impl): Java based impl of the Serverless Workflow specification. Adds support for YAML format, validation, expression evaluation, and property context.
```
Comment (cathyhongzhang) [inline workflow/spec/community/open-source.md]: The workflow-api and workflow-impl links work fine in the "Display the Rich Diff" window.

File: workflow/spec/community/readme.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Serverless Workflow Community  
+
+- [Contributors](contributors.md): the people and companies contributing to the
+  Serverless workflow specification and ecosystem
+- [Open Source](open-source.md): open source libraries that support Serverless Workflow
```
Comment (cathyhongzhang) [inline workflow/spec/community/readme.md]: The contributors and Open Source links do not work in the "Display the Rich Diff" window.

File: workflow/spec/governance/readme.md [hunk 4]
```diff
@@ -1,13 +1,9 @@
 # Serverless Workflow - Project Governance
 
-As a CNCF member project, we abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
-
 ## Review Process
 
-Given that there are only 3 [owners](owners.md) at this time the review process is as follows:
-
```
Comment (cathyhongzhang) [inline workflow/spec/governance/readme.md]: owners link does not work

File: workflow/spec/governance/readme.md [hunk 5]
```diff
@@ -23,3 +19,6 @@ Note that as our contributors count increases we hope to add more owners and rev
 If you have a technical dispute that you feel has reached an impasse with a
 subset of the community, any contributor may open an issue, specifically
 calling for a resolution vote of the current owners to resolve the dispute.
+
+## Project owners
+You can find the current list of project owners [here](owners.md).
```
Comment (cathyhongzhang) [inline workflow/spec/governance/readme.md]: owners link doe snot work

File: workflow/spec/readme.md [hunk 6]
```diff
@@ -27,20 +27,17 @@ The following documents are available:
...
 
-- [Contributors](contributors.md): people and organizations who helped
-  us get started or are actively working on the Serverless Workflow specification.
-- Coming soon: [demos & open source]() -- if you have
-  something to share about your use of Serverless Workflow, please submit a PR!
+To learn about current community efforts and how to contribute 
+reference the [Serverless Workflow Community doc](community/readme.md).
+
+As a CNCF member project, we abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+You can find the Serverless Workflow specification Governance information [here](governance/readme.md).
```
Comment (cathyhongzhang) [inline workflow/spec/readme.md]: governance link does not work

File: workflow/spec/governance/readme.md (hunk 4 above)
Comment (tsurdilo) [reply workflow/spec/governance/readme.md]: this content is removed. link no longer exists.

File: workflow/spec/readme.md (hunk 6 above)
Comment (tsurdilo) [reply workflow/spec/readme.md]: weird. this link works fine as defined. maybe issue witch diff tool

File: workflow/spec/governance/readme.md (hunk 5 above)
Comment (tsurdilo) [reply workflow/spec/governance/readme.md]: weird. this link works fine as defined. maybe issue witch diff tool

File: workflow/spec/community/readme.md (hunk 3 above)
Comment (tsurdilo) [reply workflow/spec/community/readme.md]: weird. this link works fine as defined. maybe issue witch diff tool

File: workflow/spec/community/open-source.md (hunk 2 above)
Comment (tsurdilo) [reply workflow/spec/community/open-source.md]: it seems that those link work as they define a full url. The other ones you mention dont work are relative links, but I can assure you they work fine. Again you can go to my local repo from which this pr originates and click on those links to see them work -- https://github.com/tsurdilo/wg-serverless/tree/community/workflow/spec

**Comment by cathyhongzhang** [comment]
LGTM

**Pull request merged**